gradle-tooling/src/main/java/com/liferay/blade/gradle/tooling/ProjectInfoPlugin.java

```diff
@@ -114,6 +114,12 @@ public Object buildAll(String modelName, Project project) {
 
 			String dockerContainerId = _getDockerContainerId(project);
 
+			String appServerTomcatVersion = _getAppServerTomcatVersion(project);
```

Comment:
@simonjhy   Lets not load this from the GradleTooling.  That is too slow.  We simply need to read the gradle.properties file.  That is good enough for this feature.